main.cpp

```diff
@@ -52,85 +52,85 @@ arm.rotateCounterClockwise();           // 4) demonstrates CamelCase
 
 
 //  1)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
+//  Noun:       fish
+//  action 1:   the fish swims
+fish.swim();
+//  action 2:   the fish eats algae
+fish.eatAlgae();
+//  action 3: the fish dives
+fish.dive();
 //  2)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
+//  Noun:       screen
+//  action 1:   the screen shows pictures
+screen.showPictures();
+//  action 2:   the screen flickers
+screen.flicker();
+//  action 3:   the screen turns black
+screen.turnBlack();
 //  3)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
+//  Noun:       apple
+//  action 1:   the apple provides vitamins
+apple.provideVitamins();
+//  action 2:   the apple falls from the tree
+apple.fallFromTree();
+//  action 3:   the apple causes an allergic reaction
+apple.causeAllergicReaction();
 //  4)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
+//  Noun:       water
+//  action 1:   the water quenches thirst
+water.quenchThirst();
+//  action 2:   the water makes the floor slippery
+water.makeFloorSlippery();
+//  action 3:   the water rises high
+water.riseHigh();
 //  5)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
+//  Noun:       plane
+//  action 1:   the plane flies
+plane.fly();
+//  action 2:   the plane carries passengers
+plane.carryPassengers();
+//  action 3:   the plane crashes
+plane.crash();
 //  6)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
+//  Noun:       lighter
+//  action 1:   the lighter provides fire
+lighter.provideFire();
+//  action 2:   the lighter lights a cigarette
+lighter.lightCigarette();
+//  action 3:   the lighter lights a candle
+lighter.lightCandle();
 //  7)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
+//  Noun:       calculator
+//  action 1:   the calculator solves a calculation
+calculator.solveCalculation();
+//  action 2:   the calculator helps students
+calculator.helpStudents();
+//  action 3:   requires a battery
+calculator.requireBattery();
 //  8)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
+//  Noun:       amplifier
+//  action 1:   the amplifier amplifies the guitar
+amplifier.amplifyGuitar();
+//  action 2:   the amplifier distorts the signal
+amplifier.distortSignal();
+//  action 3:   the amplifier sounds good
+amplifier.soundGood();

```

Comment:
As explained in the instructions, here is the obligatory request for some changes to get you familiar with the Pull Request Review process.
**REPLACE** the highlighted noun/action as follows:
- [ ] come up with a new noun and 3 actions
- [ ] pick a noun that needs `camelCase` for the name
- [ ] pick actions that need `camelCase` for their name.
**Do not** create an 11th noun.